types/account.go

```diff
@@ -60,7 +60,7 @@ type EthAccountI interface {
 
 // ProtoAccount defines the prototype function for BaseAccount used for an
 // AccountKeeper.
-func ProtoAccount() authtypes.AccountI {
+func ProtoAccount(_ int64) authtypes.AccountI {
```

Comment:
I think we can drop this patch from ethermint and cosmos-sdk.

It was added in v0.17.4 as an async upgrade so the default accounts would be `BaseAccount` instead of `EthAccount`, but only after the async upgrade height. It was removed in v0.19.0 and we migrated any left over accounts in the upgrade.

So removing these patches should just work as the `int64` param isn't used anymore.